Realm/RLMRealm.h

```diff
@@ -91,6 +93,26 @@ NS_ASSUME_NONNULL_BEGIN
 + (instancetype)realmWithURL:(NSURL *)fileURL;
 
 /**
+ Open a Realm asynchronously on a background queue, and then invoke a callback
```

Comment:
<s>"background queue" suggests that the main queue isn't a legal argument for this.</s>

I misunderstood what this was even referring to, and it's not wrong but maybe misleading? Maybe something like "Asynchronously open a Realm and deliver it to a block on the given queue"?